### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by JacintaNgwu**
### Description
- Add unit tests for all the methods in all the classes:
- [ ] Book
- [ ] Person
- [ ] Student
- [ ] Teacher
- [ ] Classroom
- [ ] Rental

### Team
- @utsargo
- @JacintaNgwu

### CODE REVIEW

File: spec/classroom_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require './classroom'
+
+describe Classroom do
+  classroom = Classroom.new('Math')
+  context 'should create instance of classroom class' do
+    it 'returns correct label' do
+      expect(classroom.label).to eq('Math')
+    end
```
Comment (youmari) [inline spec/classroom_spec.rb:32]: - [x] it would be better to add tests for `add_student` and also for `can_use_services` in order to cover all the methods for `can_use_services` you should test positive case and negative case

File: spec/student_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      expect(student.name).to eq('John')
+    end
+
+    it 'returns correct age' do
+      expect(student.age).to eq(18)
+    end
+
+    it 'should return correct data' do
+      expect(student.play_hooky).to eq '¯(ツ)/¯'
+    end
```
Comment (youmari) [inline spec/student_spec.rb:20]: - [x] it would be better to add tests for `add_classroom` and also for `can_use_services` in order to cover all the methods for `can_use_services` you should test positive case and negative case

File: spec/teacher_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    it 'returns correct name' do
+      expect(teacher.name).to eq('Lucas')
+    end
+
+    it 'returns correct age' do
+      expect(teacher.age).to eq(30)
+    end
+
+    it 'should return correct data' do
+      expect(teacher.can_use_services?).to eq true
```
Comment (youmari) [inline spec/teacher_spec.rb:17]: - [x] it would be better to add tests for `can_use_services` you should test positive case and negative case

File: spec/person_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      expect(person.name).to eq('John')
+    end
+
+    it 'returns correct age' do
+      expect(person.age).to eq(21)
+    end
+
+    it 'should return correct data' do
+      expect(person.can_use_services?).to eq true
+    end
```
Comment (youmari) [inline spec/person_spec.rb:17]: - [x] it would be better to add tests for `add_rental` and also for `can_use_services` in order to cover all the methods for `can_use_services` you should test the positive case and negative case

File: spec/book_spec.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+describe Book do
+  book = Book.new('Harry Potter', 'J.K. Rowling')
+  context 'should create instance of book class' do
+    it 'returns correct title' do
+      expect(book.title).to eq('Harry Potter')
+    end
+
+    it 'returns correct author' do
+      expect(book.author).to eq('J.K. Rowling')
+    end
```
Comment (youmari) [inline spec/book_spec.rb:15]: - [x] it would be better to add tests for `add_rental` and n order to cover all the methods

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @JacintaNgwu @utsargo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/classroom_spec.rb (hunk 1 above)
Comment (utsargo) [reply spec/classroom_spec.rb:32]: @youmari there is no `can_use_services` method in the `Classroom` class.

File: spec/teacher_spec.rb (hunk 3 above)
Comment (utsargo) [reply spec/teacher_spec.rb:17]: @youmari In the Teacher class `can_use_services?` will always return true, that was the project requirement. So there will be no negative case.

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @JacintaNgwu  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

File: spec/classroom_spec.rb (hunk 1 above)
Comment (youmari) [reply spec/classroom_spec.rb:32]: @utsargo sorry I just forgot to remove it
